### PR TITLE
Wording corrections

### PR DESCRIPTION
**Pull request by MartaDubno**
Improvements to wording and grammar in several docs.

### CODE REVIEW

File: src/advanced/bitcoin-anchoring.md [hunk 1]
```diff
@@ -221,8 +221,8 @@ parameters](../architecture/configuration.md) to be set.
 
 #### Bitcoind node
 
-The service uses third-party bitcoin node to communicate with the
-bitcoin blockchain network. As for Exonum v 0.1, [Bitcoin
+The service uses a third-party Bitcoin node to communicate with the
+Bitcoin blockchain network. As for Exonum v 0.1, [Bitcoin
```
Comment (slowli) [inline src/advanced/bitcoin-anchoring.md]: Slightly off-topic: propose to replace "As for Exonum v 0.1" with "Currently" in order for the article to look fresher.

File: src/architecture/consensus.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Consensus in Exonum
 
 Generally, a [consensus algorithm][wiki:consensus] is a process of
-obtaining an agreed result by a group of participants. In Exonum the
+obtaining an agreed result by a group of participants. In Exonum, the
```
[the remark this answers is not in the document]
Comment (Maria-Nosyk) [reply src/architecture/consensus.md]: Yup, I am pretty sure the comma is needless here. Commas are required only after long adverbial phrases. You will hardly ever put a comma after words "yesterday", "last year", etc. in the same cases.

File: src/advanced/bitcoin-anchoring.md [hunk 3]
```diff
@@ -444,10 +440,10 @@ Example of JSON response:
 
 - **payload.blockhash**: the hash of the anchored Exonum block
 - **payload.block_height**: the height of the anchored Exonum block
-- **content.payload.prev_tx_chain**: last tx-id of previous chain of
+- **content.payload.prev_tx_chain**: the last tx-id of the previous chain of
```
Comment (Maria-Nosyk) [inline src/advanced/bitcoin-anchoring.md]: Please, either put "." in the end of each line in the list or remove them from all the lines

File: src/advanced/bitcoin-anchoring.md [hunk 4]
```diff
@@ -456,10 +452,10 @@ GET {base_path}/actual_lect/{id}
 ```
 
 Returns the actual LECT for the specified validator, along with the
-hash of Exonum transaction published this LECT.
+hash of the Exonum transaction published in this LECT.
 
-If the specified `id` is greater or equal to validators
-amount, returns an error.
+If the specified `id` is greater or equal to validator's
```
Comment (Maria-Nosyk) [inline src/advanced/bitcoin-anchoring.md]: > If the specified `id` is greater or equal to validator's value, returns an error.

The meaning is different:
"If the specified validator `id` is greater or equal to total validators amount, an error is returned."

File: src/advanced/consensus/specification.md [hunk 5]
```diff
@@ -286,13 +286,13 @@ are placed in a separate queue (`queued`).
...
       and `state_hash`
-    - the node knows a `propose` referenced by `propose_hash`
+    - the node knows the `propose` referenced by `propose_hash`
     - the node knows all the transactions in this `propose`
 
 - Then:
 
     - Execute the proposal, if it has not yet been executed.
-    - Check that the node’s `state_hash` coincides with the `state_hash` of the
+    - Check that the node’s `state_hash` coincides with the `state_hash`
```
Comment (Maria-Nosyk) [inline src/advanced/consensus/specification.md]: > that the node’s `state_hash`

--> that `state_hash` of the node

**Pull request merged**